src/components/forms/Form/ComplexForm.stories.tsx

```diff
@@ -274,7 +274,7 @@ const Template: StoryFn<typeof Form> = (args) => {
           <PasswordInput label="Password field" />
         </Field>
         <Field
-          name={['select', 'one']}
+          name="select.one"
```

Comment:
Both versions should be valid. Why do we change this story? Will it stop working? I believe we use it in Cloud that way.

Reply:
I checked the cloud - it has never been used that way. Using array literals here doesn't make sense since we immediately transform an array into a string and use strings everywhere.

So I removed this use case. Using arrays here is no longer valid

Reply:
Yes, it's a one-way transformation just to make it easier to specify. We can remove it, but it would be nice to leave this feature.

Reply:
Yep, I missed these cases, reverted the change